### PR TITLE
[match] Update documentation

### PR DESCRIPTION
**Pull request by tedgonzalez**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

Just updating the documentation

### Description

Mention that Amazon S3 and Google Cloud are available options in the introduction

### CODE REVIEW

File: fastlane/lib/fastlane/actions/docs/sync_code_signing.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 A new approach to iOS code signing: Share one code signing identity across your development team to simplify your codesigning setup and prevent code signing issues.
 
-_match_ is the implementation of the [codesigning.guide concept](https://codesigning.guide). _match_ creates all required certificates & provisioning profiles and stores them in a separate git repository. Every team member with access to the repo can use those credentials for code signing. _match_ also automatically repairs broken and expired credentials. It's the easiest way to share signing credentials across teams
+_match_ is the implementation of the [codesigning.guide concept](https://codesigning.guide). _match_ creates all required certificates & provisioning profiles and stores them in a separate git repository, Google Cloud or Amazon S3. Every team member with access to the selected storage can use those credentials for code signing. _match_ also automatically repairs broken and expired credentials. It's the easiest way to share signing credentials across teams
```
Comment (addisonwebb) [inline fastlane/lib/fastlane/actions/docs/sync_code_signing.md]: I'm not sure if the fastlane community has a writing style guide, but I would prefer to see an Oxford comma here. e.g. `... stores them in a separate git repository, Google Cloud, or Amazon S3.`

**Review by addisonwebb — COMMENTED** [review]
This is a good clarification to the documentation. I just have one suggestion for improvement.

File: fastlane/lib/fastlane/actions/docs/sync_code_signing.md (hunk 1 above)
Comment (tedgonzalez) [reply fastlane/lib/fastlane/actions/docs/sync_code_signing.md]: Fixed!

**Review by joshdholtz — APPROVED** [review]
This for updating this! This looks like a great improvement to me ❤️

**Pull request merged**